docs/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="utf-8"/>
+    <title>keyboard-only-outlines demo</title>
+</head>
+<body>
+    <button>A button</button>
+    <input type="text"/>
+    <input type="number"/>
+    <input type="password"/>
+    <textarea></textarea>
+    <input type="radio" name="radio"/>
+    <input type="radio" name="radio"/>
+    <input type="radio" name="radio"/>
+    <br>
+    <br>
+    <button id="toggle">Toggle keyboard-only-outlines</button> <p id="state"> Current state: on</p>
+    <script type="module">
+        import keyboardOnlyOutlines from '../index.js';
+
+        let tearDown = keyboardOnlyOutlines();
```

Comment:
Prefer `const`  over `let` if the values don't change.

Reply:
It does change. Clicking the button can cause that variable to be changed.

Reply:
I mean there is the `state` and `toggleButton` that doesn't change. Also prepend them with `Node` so that we know that they are node elements.